backend/src/components/permissionUtils.js

```diff
@@ -154,6 +154,31 @@ function checkSdcSchoolCollectionAccess(req, res, next) {
   }
 }
 
+function loadInstituteCollection(req, res, next) {
+  if (req.session.activeInstituteType === 'SCHOOL') {
+    return loadSdcSchoolCollection(req, res, next);
+  } else {
+    return loadSdcDistrictCollection(req, res, next);
+  }
+}
+
+function checkInstituteCollectionAccess(req, res, next) {
+  if (req.session.activeInstituteType === 'SCHOOL') {
+    return checkSdcSchoolCollectionAccess(req, res, next);
+  } else {
+    return checkSdcDistrictCollectionAccess(req, res, next);
+  }
+}
+
+function checkIfCreateorUpdateIsAllowed(req, res, next) {
```

Comment:
Can we rename this function to be more SDC specific?

Reply:
fixed